indra/sources/tees/parse_tees.py

```diff
@@ -257,14 +256,18 @@ def index_to_sentence(self, index):
             return None
 
 
-def parse_tees_output_directory(output_dir):
-    """Parses the files in the TEES output directory. And returns a networkx
-    graph.
+def parse_tees_output_files(a1_text, a2_text, sentence_segmentations):
```

Comment:
Here `tees` is not needed in the name. It also looks like these aren't actually files but rather strings so the function could be named `parse_output`.